platform-sdk/platform-apps/tests/ConsistencyTestingTool/src/test/java/com/swirlds/demo/consistency/ConsistencyTestingToolRoundTests.java

```diff
@@ -32,6 +34,7 @@
 import com.swirlds.platform.system.address.AddressBook;
 import com.swirlds.platform.system.transaction.ConsensusTransaction;
 import com.swirlds.platform.system.transaction.ConsensusTransactionImpl;
+import com.swirlds.proto.event.EventPayload.PayloadOneOfType;
```

Comment:
Where does this package name come from? I think the correct way to import this might be like this:

```
import com.hedera.hapi.platform.event.EventPayload.PayloadOneOfType;
```

Reply:
it is generated based on the `.proto` file:
```
option java_package = "com.swirlds.proto.legacy.event";
// <<<pbj.java_package = "com.swirlds.proto.event">>> This comment is special code for setting PBJ Compiler java package
```